go/interpreter/lfvm/interpreter.go

```diff
@@ -243,30 +243,12 @@ func steps(c *context, oneStepOnly bool) {
 
 		op := c.code[c.pc].opcode
 
-		// JUMP_TO is an LFVM specific operation that has no gas costs nor stack usage.
-		if c.code[c.pc].opcode == JUMP_TO {
```

Comment:
This check comes as a surprise to me, but I guess it makes sense, it would tax every other instruction and behaviour is trivial.